lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -91,6 +91,8 @@ static SpecialFunctionHandler::HandlerInfo handlerInfo[] = {
   add("klee_define_fixed_object", handleDefineFixedObject, false),
   add("klee_get_obj_size", handleGetObjSize, true),
   add("klee_get_errno", handleGetErrno, true),
+  add("__errno_location", handleErrnoLocation, true),
+  add("__error", handleErrnoLocation, true),
```

Comment:
In GNU/Linux `error()` has 4 arguments:
```C
#include <error.h>
void error(int status, int errnum, const char *format, ...);
```

This means coreutils fail with:
```
Assertion `arguments.size() == 0 && "invalid number of arguments to __errno_location"' failed.
```
Bitcode:
```llvm
@error = weak alias void (i32, i32, i8*, ...)* @__error
declare hidden void @__error(i32, i32, i8*, ...) #2
```